apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -107,11 +108,7 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
     );
   };
 
-  const { shouldLockDisableProps } = useLockedFieldsManager(
-    eventType,
-    t("locked_fields_admin_description"),
-    t("locked_fields_member_description")
-  );
+  const { shouldLockDisableProps } = useLockedFieldsManager(eventType, formMethods, t);
```

Comment:
We now need form methods in order to set discretely the different fields in the event type metadata